packages/wonder-blocks-popover/src/components/focus-manager.tsx

```diff
@@ -86,6 +87,11 @@ export default class FocusManager extends React.Component<Props> {
      */
     focusableElementsInPopover: Array<HTMLElement> = [];
```

Comment:
The name of this field is a little confusing because we're setting these elements to be non-focusable when the focus exits the popover or whatever.  

**suggestion:** consider renaming this to something like `elementsThatCanBeFocusable` or something like that to avoid confusion as to whether the elements in the array are currently focusable or not.